src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
> Isn't that a different serialization mechanism?

Right it is different type of serialization but the root cause is same which is they expect same field names of the classes which we are discussing here. it will be strange to support DCS on cross-platforms and not supporting binary serialization. I think if we decided to support DCS cross-plat then the work needed there should be similar or close to what need to be done for binary serialization.